### ISSUE TITLE
Search field shortcut breaks EMACS bindings

### ISSUE THREAD

**Issue by zanshin** [RocketChat/Rocket.Chat#6374]
Rocket.Chat Version: 0.53.0
Running Instances: 1
DB Replicaset OpLog: Disabled ?
Node Version: v4.5.0

Using ctrl-k as the keyboard shortcut for the search field breaks using EMACS keyboard bindings for editing within the message text box. A search for "emacs text editing shortcuts" will reveal any number of results detailing these widely used key combinations, e.g., https://msdn.microsoft.com/en-us/library/ms165528(v=vs.90).aspx

This is not a question of "changing the default text" shown in the search box, it is a question of changing the keyboard combination to something else. "F" is typically used for search (find) in applications. 

I am opening this issue since #6220 was closed before I had a change to comment and now it is being ignored.

**Comment by nmushegian** [comment]
THIS ~ just came here to say this ~ this is why I searched rocket chat issues ~ plus ONE ~ #hotkeys #computercommands

**Comment by livnev** [comment]
I would really appreciate it if the search shortcut ctrl-k was remapped to something else, or if there was an option in the settings for remapping it.

**Comment by eengstrom** [comment]
Happy to see that @rodrigok marked this as planned, but where is the design or discussion for the new features being discussed?

**Comment by slowder** [comment]
Agreed, this is very annoying. I've been on RC for 3 months now and I still hit ctrl-k all the time.

**Comment by ktims** [comment]
Please avoid overriding existing global browser shortcuts like Ctrl-F and Ctrl-K. It's quite irritating having to choose between blacklisting the app's ability to override shortcuts or losing one shortcut or the other.
